### ISSUE TITLE
Broken Pipe HELL

### ISSUE THREAD

**Issue by wrideout** [subecho/KarmaChameleon#46]
```
54.167.217.233 - - [07/Apr/2021 23:46:34] "POST /slack/events HTTP/1.1" 200 -
3.80.156.142 - - [08/Apr/2021 00:19:47] "POST /slack/events HTTP/1.1" 200 -
----------------------------------------
Exception happened during processing of request from ('74.120.14.38', 58126)
Traceback (most recent call last):
  File "/usr/lib/python3.8/socketserver.py", line 316, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/usr/lib/python3.8/socketserver.py", line 347, in process_request
    self.finish_request(request, client_address)
  File "/usr/lib/python3.8/socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.8/http/server.py", line 647, in __init__
    super().__init__(*args, **kwargs)
  File "/usr/lib/python3.8/socketserver.py", line 720, in __init__
    self.handle()
  File "/usr/lib/python3.8/http/server.py", line 427, in handle
    self.handle_one_request()
  File "/usr/lib/python3.8/http/server.py", line 395, in handle_one_request
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib/python3.8/socket.py", line 669, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 104] Connection reset by peer
----------------------------------------
74.120.14.38 - - [08/Apr/2021 01:26:44] code 400, message Bad request version ('À\x14À')
74.120.14.38 - - [08/Apr/2021 01:26:44] "{whZú¸
                                                °Þ#@+òcëPÎô¢vÃl²ÌÛ|ïÀ/À+ÀÀÀÀ        ÀÀ" 400 -
74.120.14.38 - - [08/Apr/2021 01:26:46] "GET / HTTP/1.1" 404 -
74.120.14.38 - - [08/Apr/2021 01:26:49] "GET / HTTP/1.1" 404 -
35.175.200.162 - - [08/Apr/2021 02:38:20] "POST /slack/events HTTP/1.1" 200 -
```

**Comment by wrideout** [comment]
18.213.218.253 - - [21/Apr/2021 21:27:00] "POST /slack/events HTTP/1.1" 200 -
3.235.226.122 - - [21/Apr/2021 21:27:01] "POST /slack/events HTTP/1.1" 200 -
3.237.198.252 - - [21/Apr/2021 21:28:44] "POST /slack/events HTTP/1.1" 200 -
54.167.233.34 - - [21/Apr/2021 21:39:35] "POST /slack/events HTTP/1.1" 200 -
35.168.23.239 - - [21/Apr/2021 21:39:41] "POST /slack/events HTTP/1.1" 200 -
125.64.94.136 - - [21/Apr/2021 21:45:42] "GET / HTTP/1.0" 404 -
125.64.94.136 - - [21/Apr/2021 21:45:43] "GET /favicon.ico HTTP/1.1" 404 -
125.64.94.136 - - [21/Apr/2021 21:45:43] "GET /robots.txt HTTP/1.1" 404 -
125.64.94.136 - - [21/Apr/2021 21:45:44] "GET /.well-known/security.txt HTTP/1.1" 404 -
128.199.38.218 - - [21/Apr/2021 23:00:10] "HEAD / HTTP/1.0" 200 -
128.199.38.218 - - [21/Apr/2021 23:00:11] "GET /system_api.php HTTP/1.1" 404 -
128.199.38.218 - - [21/Apr/2021 23:00:11] "GET /system_api.php HTTP/1.1" 404 -
128.199.38.218 - - [21/Apr/2021 23:00:11] "GET /c/version.js HTTP/1.1" 404 -
128.199.38.218 - - [21/Apr/2021 23:00:11] "GET /streaming/clients_live.php HTTP/1.1" 404 -
128.199.38.218 - - [21/Apr/2021 23:00:12] "GET /stalker_portal/c/version.js HTTP/1.1" 404 -
128.199.38.218 - - [21/Apr/2021 23:00:12] "GET /client_area/ HTTP/1.1" 404 -
128.199.38.218 - - [21/Apr/2021 23:00:12] "GET /stalker_portal/c/ HTTP/1.1" 404 -
128.199.38.218 - - [21/Apr/2021 23:00:12] "GET /stream/live.php HTTP/1.1" 404 -
34.238.137.70 - - [22/Apr/2021 00:17:39] "POST /slack/events HTTP/1.1" 200 -
34.229.46.127 - - [22/Apr/2021 00:17:40] "POST /slack/events HTTP/1.1" 200 -
52.205.248.253 - - [22/Apr/2021 00:18:35] "POST /slack/events HTTP/1.1" 200 -
3.237.14.92 - - [22/Apr/2021 00:18:44] "POST /slack/events HTTP/1.1" 200 -
3.236.170.216 - - [22/Apr/2021 00:19:42] "POST /slack/events HTTP/1.1" 200 -
34.205.90.210 - - [22/Apr/2021 00:19:46] "POST /slack/events HTTP/1.1" 200 -
45.155.205.27 - - [22/Apr/2021 04:44:35] code 400, message Bad request syntax ('\x02\x01\x00\x00\x00\x00\x00\x06build')
45.155.205.27 - - [22/Apr/2021 04:44:35] "build" 400 -
45.155.205.27 - - [22/Apr/2021 04:44:35] code 400, message Bad request syntax ('\x02\x01\x00\x00\x00\x00\x00\x06build')
45.155.205.27 - - [22/Apr/2021 04:44:35] "build" 400 -
34.231.244.53 - - [22/Apr/2021 14:00:12] "POST /slack/events HTTP/1.1" 200 -
3.215.142.218 - - [22/Apr/2021 14:16:57] "POST /slack/events HTTP/1.1" 200 -
54.210.184.190 - - [22/Apr/2021 14:21:20] "POST /slack/events HTTP/1.1" 200 -
34.238.137.70 - - [22/Apr/2021 14:25:33] "POST /slack/events HTTP/1.1" 200 -
3.239.160.228 - - [22/Apr/2021 14:32:21] "POST /slack/events HTTP/1.1" 200 -
23.23.1.22 - - [22/Apr/2021 14:36:37] "POST /slack/events HTTP/1.1" 200 -
54.162.122.230 - - [22/Apr/2021 14:43:38] "POST /slack/events HTTP/1.1" 200 -
3.229.122.7 - - [22/Apr/2021 14:43:39] "POST /slack/events HTTP/1.1" 200 -
3.82.221.84 - - [22/Apr/2021 14:43:39] "POST /slack/events HTTP/1.1" 200 -
44.192.128.152 - - [22/Apr/2021 14:44:05] "POST /slack/events HTTP/1.1" 200 -
54.80.35.197 - - [22/Apr/2021 14:44:36] "POST /slack/events HTTP/1.1" 200 -
35.168.23.239 - - [22/Apr/2021 14:44:50] "POST /slack/events HTTP/1.1" 200 -
3.85.162.39 - - [22/Apr/2021 14:44:57] "POST /slack/events HTTP/1.1" 200 -
34.229.46.127 - - [22/Apr/2021 14:45:25] "POST /slack/events HTTP/1.1" 200 -
3.215.142.218 - - [22/Apr/2021 14:45:56] "POST /slack/events HTTP/1.1" 200 -
34.231.244.53 - - [22/Apr/2021 14:47:33] "POST /slack/events HTTP/1.1" 200 -
3.89.99.149 - - [22/Apr/2021 14:47:34] "POST /slack/events HTTP/1.1" 200 -
3.82.64.26 - - [22/Apr/2021 14:48:29] "POST /slack/events HTTP/1.1" 200 -
3.82.64.26 - - [22/Apr/2021 14:49:20] "POST /slack/events HTTP/1.1" 200 -
18.213.218.253 - - [22/Apr/2021 14:49:21] "POST /slack/events HTTP/1.1" 200 -
91.241.19.60 - - [22/Apr/2021 16:30:10] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
91.241.19.60 - - [22/Apr/2021 16:30:10] "/*àCookie: mstshash=Administr" 400 -
34.229.46.127 - - [22/Apr/2021 17:31:05] "POST /slack/events HTTP/1.1" 200 -
34.238.137.70 - - [22/Apr/2021 17:38:31] "POST /slack/events HTTP/1.1" 200 -
54.80.35.197 - - [22/Apr/2021 17:38:50] "POST /slack/events HTTP/1.1" 200 -
34.238.137.70 - - [22/Apr/2021 17:40:37] "POST /slack/events HTTP/1.1" 200 -
18.209.229.197 - - [22/Apr/2021 17:41:13] "POST /slack/events HTTP/1.1" 200 -
3.85.169.81 - - [22/Apr/2021 17:41:40] "POST /slack/events HTTP/1.1" 200 -
54.90.68.20 - - [22/Apr/2021 17:41:41] "POST /slack/events HTTP/1.1" 200 -
18.208.182.185 - - [22/Apr/2021 18:24:41] "POST /slack/events HTTP/1.1" 200 -
52.205.62.103 - - [22/Apr/2021 18:25:17] "POST /slack/events HTTP/1.1" 200 -
54.167.233.34 - - [22/Apr/2021 18:25:17] "POST /slack/events HTTP/1.1" 200 -
52.205.248.253 - - [22/Apr/2021 19:09:03] "POST /slack/events HTTP/1.1" 200 -
44.192.91.13 - - [22/Apr/2021 19:09:15] "POST /slack/events HTTP/1.1" 200 -
3.84.1.25 - - [22/Apr/2021 19:09:21] "POST /slack/events HTTP/1.1" 200 -
34.237.53.27 - - [22/Apr/2021 19:09:33] "POST /slack/events HTTP/1.1" 200 -
3.84.1.25 - - [22/Apr/2021 19:09:50] "POST /slack/events HTTP/1.1" 200 -
3.89.99.149 - - [22/Apr/2021 19:11:26] "POST /slack/events HTTP/1.1" 200 -
54.167.188.20 - - [22/Apr/2021 19:12:18] "POST /slack/events HTTP/1.1" 200 -
54.80.35.197 - - [22/Apr/2021 19:12:31] "POST /slack/events HTTP/1.1" 200 -
18.209.229.197 - - [22/Apr/2021 19:13:52] "POST /slack/events HTTP/1.1" 200 -
3.90.251.108 - - [22/Apr/2021 19:14:08] "POST /slack/events HTTP/1.1" 200 -
54.167.188.20 - - [22/Apr/2021 19:14:17] "POST /slack/events HTTP/1.1" 200 -
52.90.61.239 - - [22/Apr/2021 19:14:20] "POST /slack/events HTTP/1.1" 200 -
54.172.248.40 - - [22/Apr/2021 19:15:07] "POST /slack/events HTTP/1.1" 200 -
18.212.106.242 - - [22/Apr/2021 19:15:10] "POST /slack/events HTTP/1.1" 200 -
3.235.226.122 - - [22/Apr/2021 19:15:10] "POST /slack/events HTTP/1.1" 200 -
18.234.225.56 - - [22/Apr/2021 19:46:23] "POST /slack/events HTTP/1.1" 200 -
3.87.136.30 - - [22/Apr/2021 19:48:10] "POST /slack/events HTTP/1.1" 200 -
3.84.1.25 - - [22/Apr/2021 19:48:15] "POST /slack/events HTTP/1.1" 200 -
54.210.184.190 - - [22/Apr/2021 19:48:22] "POST /slack/events HTTP/1.1" 200 -
3.236.45.98 - - [22/Apr/2021 19:48:30] "POST /slack/events HTTP/1.1" 200 -
54.86.242.132 - - [22/Apr/2021 19:48:49] "POST /slack/events HTTP/1.1" 200 -
34.231.244.53 - - [22/Apr/2021 19:49:23] "POST /slack/events HTTP/1.1" 200 -
92.118.160.61 - - [22/Apr/2021 20:36:15] "GET / HTTP/1.1" 404 -
34.228.230.3 - - [22/Apr/2021 22:13:16] "POST /slack/events HTTP/1.1" 200 -
54.172.248.40 - - [22/Apr/2021 22:13:16] "POST /slack/events HTTP/1.1" 200 -
3.230.123.119 - - [22/Apr/2021 22:26:39] "POST /slack/events HTTP/1.1" 200 -
45.155.205.27 - - [22/Apr/2021 22:30:12] code 400, message Bad request syntax ('\x02\x01\x00\x00\x00\x00\x00\x06build')
45.155.205.27 - - [22/Apr/2021 22:30:12] "build" 400 -
45.155.205.27 - - [22/Apr/2021 22:30:12] code 400, message Bad request syntax ('\x02\x01\x00\x00\x00\x00\x00\x06build')
45.155.205.27 - - [22/Apr/2021 22:30:12] "build" 400 -
92.118.161.29 - - [23/Apr/2021 01:49:52] "GET / HTTP/1.0" 404 -
3.236.45.98 - - [23/Apr/2021 02:39:45] "POST /slack/events HTTP/1.1" 200 -
3.82.64.26 - - [23/Apr/2021 02:39:47] "POST /slack/events HTTP/1.1" 200 -
123.160.221.24 - - [23/Apr/2021 04:02:55] code 400, message Bad request version ('\x00/\x00')
123.160.221.24 - - [23/Apr/2021 04:02:55] "ieU§ärandom1random2random3random4
                                                                            /" 400 -
123.160.221.24 - - [23/Apr/2021 04:02:56] code 400, message Bad request syntax ('\x16\x03\x00\x00S\x01\x00\x00O\x03\x00?G×÷º,îê²`~ó\x00ý\x82{¹Õ\x96Èw\x9bæÄÛ<=Ûoï\x10n\x00\x00(\x00\x16\x00\x13\x00')
123.160.221.24 - - [23/Apr/2021 04:02:56] "SO?G×÷º,îê²`~óý{¹ÕÈwæÄÛ<=Ûoïn(" 400 -
123.160.221.24 - - [23/Apr/2021 04:02:56] "GET / HTTP/1.0" 404 -
123.160.221.24 - - [23/Apr/2021 04:02:57] "HEAD / HTTP/1.1" 200 -
123.160.221.24 - - [23/Apr/2021 04:02:58] "GET / HTTP/1.1" 404 -
123.160.221.24 - - [23/Apr/2021 04:02:59] "GET /favicon.ico HTTP/1.1" 404 -
123.160.221.24 - - [23/Apr/2021 04:03:00] "HEAD / HTTP/1.1" 200 -
123.160.221.24 - - [23/Apr/2021 04:03:01] "GET / HTTP/1.1" 404 -
123.160.221.24 - - [23/Apr/2021 04:03:01] "GET /favicon.ico HTTP/1.1" 404 -
123.160.221.24 - - [23/Apr/2021 04:03:01] "HEAD / HTTP/1.1" 200 -
123.160.221.24 - - [23/Apr/2021 04:03:03] "GET / HTTP/1.1" 404 -
123.160.221.24 - - [23/Apr/2021 04:03:03] "GET /favicon.ico HTTP/1.1" 404 -
34.228.230.3 - - [23/Apr/2021 13:33:19] "POST /slack/events HTTP/1.1" 200 -
54.209.54.223 - - [23/Apr/2021 14:02:11] "POST /slack/events HTTP/1.1" 200 -
18.213.218.253 - - [23/Apr/2021 14:31:45] "POST /slack/events HTTP/1.1" 200 -
35.174.60.127 - - [23/Apr/2021 14:31:56] "POST /slack/events HTTP/1.1" 200 -
54.92.184.209 - - [23/Apr/2021 14:32:56] "POST /slack/events HTTP/1.1" 200 -
52.87.242.53 - - [23/Apr/2021 14:34:10] "POST /slack/events HTTP/1.1" 200 -
3.238.73.141 - - [23/Apr/2021 15:45:29] "POST /slack/events HTTP/1.1" 200 -
3.236.45.98 - - [23/Apr/2021 15:46:20] "POST /slack/events HTTP/1.1" 200 -
35.168.23.239 - - [23/Apr/2021 15:46:21] "POST /slack/events HTTP/1.1" 200 -
79.124.62.254 - - [23/Apr/2021 16:05:36] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
79.124.62.254 - - [23/Apr/2021 16:05:36] "/*àCookie: mstshash=Administr" 400 -
45.155.205.27 - - [23/Apr/2021 16:22:21] code 400, message Bad request syntax ('\x02\x01\x00\x00\x00\x00\x00\x06build')
45.155.205.27 - - [23/Apr/2021 16:22:21] "build" 400 -
45.155.205.27 - - [23/Apr/2021 16:22:21] code 400, message Bad request syntax ('\x02\x01\x00\x00\x00\x00\x00\x06build')
45.155.205.27 - - [23/Apr/2021 16:22:21] "build" 400 -
34.228.11.143 - - [23/Apr/2021 19:03:27] "POST /slack/events HTTP/1.1" 200 -
54.167.233.34 - - [23/Apr/2021 19:03:49] "POST /slack/events HTTP/1.1" 200 -
3.239.160.228 - - [23/Apr/2021 19:04:23] "POST /slack/events HTTP/1.1" 200 -
3.236.45.98 - - [23/Apr/2021 19:04:32] "POST /slack/events HTTP/1.1" 200 -
34.237.53.27 - - [23/Apr/2021 19:08:36] "POST /slack/events HTTP/1.1" 200 -
34.228.230.3 - - [23/Apr/2021 19:08:48] "POST /slack/events HTTP/1.1" 200 -
3.232.96.110 - - [23/Apr/2021 19:13:13] "POST /slack/events HTTP/1.1" 200 -
54.167.188.20 - - [23/Apr/2021 19:14:34] "POST /slack/events HTTP/1.1" 200 -
18.234.225.56 - - [23/Apr/2021 19:14:36] "POST /slack/events HTTP/1.1" 200 -
34.204.176.196 - - [23/Apr/2021 19:14:36] "POST /slack/events HTTP/1.1" 200 -
52.207.236.208 - - [23/Apr/2021 19:15:10] "POST /slack/events HTTP/1.1" 200 -
3.82.64.26 - - [23/Apr/2021 19:33:12] "POST /slack/events HTTP/1.1" 200 -
54.208.206.45 - - [23/Apr/2021 19:34:58] "POST /slack/events HTTP/1.1" 200 -
52.91.33.106 - - [23/Apr/2021 21:00:24] "POST /slack/events HTTP/1.1" 200 -
34.204.176.196 - - [23/Apr/2021 21:01:17] "POST /slack/events HTTP/1.1" 200 -
3.215.142.218 - - [23/Apr/2021 21:01:18] "POST /slack/events HTTP/1.1" 200 -
3.229.122.7 - - [23/Apr/2021 21:01:21] "POST /slack/events HTTP/1.1" 200 -
54.172.248.40 - - [23/Apr/2021 22:19:51] "POST /slack/events HTTP/1.1" 200 -
18.234.225.56 - - [23/Apr/2021 22:20:04] "POST /slack/events HTTP/1.1" 200 -
91.241.19.99 - - [23/Apr/2021 23:21:10] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
91.241.19.99 - - [23/Apr/2021 23:21:10] "/*àCookie: mstshash=Administr" 400 -
91.241.19.99 - - [23/Apr/2021 23:49:04] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
91.241.19.99 - - [23/Apr/2021 23:49:04] "/*àCookie: mstshash=Administr" 400 -
23.129.64.245 - - [24/Apr/2021 00:07:10] "GET / HTTP/1.1" 404 -
89.248.165.52 - - [24/Apr/2021 03:21:21] code 400, message Bad request syntax ('\x04\x01\x00\x19h/\x00!\x00')
89.248.165.52 - - [24/Apr/2021 03:21:21] "h/!" 400 -
----------------------------------------
Exception happened during processing of request from ('89.248.165.52', 52070)
Traceback (most recent call last):
  File "/usr/lib/python3.8/socketserver.py", line 316, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/usr/lib/python3.8/socketserver.py", line 347, in process_request
    self.finish_request(request, client_address)
  File "/usr/lib/python3.8/socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.8/http/server.py", line 647, in __init__
    super().__init__(*args, **kwargs)
  File "/usr/lib/python3.8/socketserver.py", line 720, in __init__
    self.handle()
  File "/usr/lib/python3.8/http/server.py", line 427, in handle
    self.handle_one_request()
  File "/usr/lib/python3.8/http/server.py", line 405, in handle_one_request
    if not self.parse_request():
  File "/usr/lib/python3.8/http/server.py", line 321, in parse_request
    self.send_error(
  File "/usr/lib/python3.8/http/server.py", line 482, in send_error
    self.wfile.write(body)
  File "/usr/lib/python3.8/socketserver.py", line 799, in write
    self._sock.sendall(b)
BrokenPipeError: [Errno 32] Broken pipe
----------------------------------------
89.248.165.52 - - [24/Apr/2021 03:22:21] code 400, message Bad request syntax ('\x05\x01\x00')
89.248.165.52 - - [24/Apr/2021 03:22:21] "" 400 -
----------------------------------------
Exception happened during processing of request from ('89.248.165.52', 56002)
Traceback (most recent call last):
  File "/usr/lib/python3.8/socketserver.py", line 316, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/usr/lib/python3.8/socketserver.py", line 347, in process_request
    self.finish_request(request, client_address)
  File "/usr/lib/python3.8/socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.8/http/server.py", line 647, in __init__
    super().__init__(*args, **kwargs)
  File "/usr/lib/python3.8/socketserver.py", line 720, in __init__
    self.handle()
  File "/usr/lib/python3.8/http/server.py", line 427, in handle
    self.handle_one_request()
  File "/usr/lib/python3.8/http/server.py", line 405, in handle_one_request
    if not self.parse_request():
  File "/usr/lib/python3.8/http/server.py", line 321, in parse_request
    self.send_error(
  File "/usr/lib/python3.8/http/server.py", line 482, in send_error
    self.wfile.write(body)
  File "/usr/lib/python3.8/socketserver.py", line 799, in write
    self._sock.sendall(b)
BrokenPipeError: [Errno 32] Broken pipe
----------------------------------------
89.248.165.52 - - [24/Apr/2021 03:22:21] code 501, message Unsupported method ('CONNECT')
89.248.165.52 - - [24/Apr/2021 03:22:21] "CONNECT hotmail-com.olc.protection.outlook.com:25 HTTP/1.1" 501 -
79.124.62.254 - - [24/Apr/2021 03:26:50] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
79.124.62.254 - - [24/Apr/2021 03:26:50] "/*àCookie: mstshash=Administr" 400 -
^CTraceback (most recent call last):
  File "karma_chameleon/main.py", line 135, in <module>
    app.start(port=int(os.environ.get("PORT", 3000)))
  File "/home/karma/KarmaChameleon/.venv/lib/python3.8/site-packages/slack_bolt/app/app.py", line 322, in start
    self._development_server.start()
  File "/home/karma/KarmaChameleon/.venv/lib/python3.8/site-packages/slack_bolt/app/app.py", line 1005, in start
    self._server.serve_forever(0.05)
  File "/usr/lib/python3.8/socketserver.py", line 237, in serve_forever
    self._handle_request_noblock()
  File "/usr/lib/python3.8/socketserver.py", line 316, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/usr/lib/python3.8/socketserver.py", line 347, in process_request
    self.finish_request(request, client_address)
  File "/usr/lib/python3.8/socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.8/http/server.py", line 647, in __init__
    super().__init__(*args, **kwargs)
  File "/usr/lib/python3.8/socketserver.py", line 720, in __init__
    self.handle()
  File "/usr/lib/python3.8/http/server.py", line 427, in handle
    self.handle_one_request()
  File "/usr/lib/python3.8/http/server.py", line 395, in handle_one_request
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib/python3.8/socket.py", line 669, in readinto
    return self._sock.recv_into(b)
KeyboardInterrupt

**Comment by wrideout** [comment]
Consider using nginx+gunicorn?

**Comment by wrideout** [comment]
This is fixed and superseded by the closure of #49
